cmd/kyma/apply/function/function.go

```diff
@@ -103,32 +105,45 @@ func (c *command) Run() error {
 		return err
 	}
 
-	subscriptions, err := resources.NewSubscriptions(configuration)
-	if err != nil {
-		step.Failure()
-		return err
-	}
+	operators := []operator.Operator{}
 
-	kymaAddress, err := c.kymaHostAddress()
```

Comment:
So we totally remove this functionality? Am I right?

Reply:
Yes. API rules do not require to pass the full runtime domain. The controller is assuming it based on kyma gateway internally